governance/handler.go

```diff
@@ -422,3 +422,7 @@ func (gov *Governance) GetGovernanceItemAtNumber(num uint64, key string) (interf
 		return nil, ErrItemNotFound
 	}
 }
+
+func (gov *Governance) GetItemAtNumberByIntKey(num uint64, key int) (interface{}, error) {
```

Comment:
Description comment is missing.

Reply:
thanks for your comment. I'll do it in a next pr